packages/core/src/generatePolicy.js

```diff
@@ -297,9 +339,20 @@ function createModuleInspector(opts) {
     }
     // agregate globals
     if (!packageToGlobals.has(packageName)) {
-      packageToGlobals.set(packageName, [])
+      packageToGlobals.set(packageName, new Map())
     }
     let packageGlobals = packageToGlobals.get(packageName)
+
+    // XXX temporary fix while we do not have writable global property
+    // support at runtime
+    // https://github.com/LavaMoat/LavaMoat/issues/1554
+    for (const [path, value] of [...packageGlobals, ...foundGlobals]) {
+      const pathParts = path.split('.')
+      if (pathParts.length > 1 && value === POLICY_WRITE) {
+        packageGlobals.set(pathParts[0], true)
+      }
```

Comment:
This makes the actual result be more open than what the user has defined. I'm not sure that's what we should be doing.

What circumstances are you trying to handle here? policy with "a.b.c":"write" being generated or an entry in policy-overrides that does it?
If the latter, we should throw. If the former, I recall we were going to prevent nested writes from being generated.

Reply:
brain fart